docs/_ext/single_sourced_data.py

```diff
@@ -76,6 +76,11 @@ def _nodes_from_rst(
     """Turn an RST string into a list of nodes.
 
     These nodes can be used in the document.
+
+    :param state: The superclass for a docutils statemachine which contains a list or transitions
```

Comment:
Technically, it's not a class but an instance that is initialized and managed by Sphinx. Back when I had to figure this out, it was not obvious, it's good that you're documenting this.